components/form/form.js

```diff
@@ -58,7 +59,13 @@ class Form extends FormMixin(LitElement) {
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
-		this.addEventListener('d2l-dialog-close', () => {
+		const dialogAncestor = findComposedAncestor(
+			this,
+			(node) => { return (node?.tagName?.includes('D2L-DIALOG')); }
```

Comment:
The `.includes(` should cover that?

Reply:
Oh good point. I glossed over that. So it would just be anyone (like LCS) that might have extended the mixin for their own component. Maybe they're on their own to reset their forms.

Reply:
Wonder if something like `this.prototype instanceof DialogMixin` would work?

Reply:
Still looking into it but so far haven't been able to get that to work

Reply:
I know in a couple other places we've set a property that can be checked, which seems kinda hacky. What Dave proposed seems much nicer.

Reply:
I haven't been able to get anything along the lines of the `node.prototype instanceof DialogMixin` check working. I can get setting a property on dialog mixin and checking for that working. Do we want to go through with the property option? Any other ideas that I maybe haven't tried?

Reply:
Hmm, I feel like a `_isDialogMixin = true` "private" variable would be slightly more resilient than the tag name check, but am good with either.